### PR TITLE
confectionery: update livecheck

### PR DESCRIPTION
**Pull request by daeho-ro**
Same as lotus case :)

---

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Review by bevanjkay — COMMENTED** [review]
Should be able to use the same as `lotus` here.
https://app-updates.vadimdemedes.com/confectionery/download/Confectionery-1.2.1-mac.dmg

**Comment by daeho-ro** [comment]
@bevanjkay I missed that one, will be implemented soon.

File: Casks/c/confectionery.rb [hunk 1]
```diff
@@ -2,14 +2,17 @@
   version "1.2.1"
   sha256 :no_check
```
Comment (bevanjkay) [inline Casks/c/confectionery.rb:3]: We can add a checksum now too, all good to wait for the CI failure to reveal what it is if that's easier.

**Review by bevanjkay — APPROVED** [review]

**Comment by daeho-ro** [comment]
Is this url is slower than before? Maybe cdn problem or vercel? Hm...

**Comment by khipp** [comment]
@daeho-ro This might be an upstream issue. I'm getting a connection timeout when trying to download the file directly from the website (https://confectioneryapp.com/download).

**Pull request merged**